app/cothorityd/cothorityd.go

```diff
@@ -1,127 +1,400 @@
-/*
```

Comment:
You should add some documentation to the cothorityd, so it shows up nicely in `godoc`.